js/core/utils/size.js

```diff
@@ -36,14 +36,18 @@ const getBoxSizingOffset = function(name, elementStyles, boxParams) {
 
     return 0;
 };
+const getComputedStyle = function(element) {
```

Comment:
The name is the same as window method (https://developer.mozilla.org/ru/docs/Web/API/Window/getComputedStyle).
I recommend to rename it to avoid confusion.